EKF/ekf_helper.cpp

```diff
@@ -750,13 +750,13 @@ bool Ekf::resetMagHeading(Vector3f &mag_init, bool increase_yaw_var, bool update
 	if (update_buffer) {
 		// add the reset amount to the output observer buffered data
 		for (uint8_t i = 0; i < _output_buffer.get_length(); i++) {
-			// Note q1 *= q2 is equivalent to q1 = q2 * q1
+			// Note q1 *= q2 is equivalent to q1 = q1 * q2
```

Comment:
Suggest we remove this line and change line 754 to:

_output_buffer[i].quat_nominal = _state_reset_status.quat_change * _output_buffer[i].quat_nominal;

to make the use of the eigen library order convention more obvious.

Reply:
Suggest we remove this comment line, fix the sign in calculation of _state_reset_status.quat_change and discontinue use of the *= operator